src/StarterSite.php

```diff
@@ -469,4 +470,35 @@ public function update_twig_environment_options( $options ) {
 
 		return $options;
 	}
+
+	/**
+	 * Take a hierarchical WP menu and flatten it.
+	 *
+	 * This is useful when a menu doesn't allow for sub-items. It just makes all
+	 * items top-level, in the order they appear in the admin UX. Thus, if a
+	 * site editor accidentally assigns a sub-item on one of these menus (due to
+	 * either not knowing what is allowed, or a dragging error, etc.) then the
+	 * menu item will still behave in an expected way.
+	 *
+	 * @param object $menu The menu object to be flattened.
+	 * @param array  $items The current set of child menu links to flatten in.
+	 *
+	 * @return void
+	 */
+	private function flatten_menu( &$menu, $items = false ) {
```

Comment:
This code crashes when the menu passed to it doesn't exist (e.g. new site install where the menu hasn't been created yet)
```suggestion
	private function flatten_menu( &$menu, $items = false ) {
	if ( ! $menu ) {
	  return;
	}
```